ETH-Merge-Bots/Milestones-in-TTD/README.md

```diff
@@ -0,0 +1,75 @@
+# Milestones in Terminal Total Difficulty
+
+## Description
+
+This bot monitors `totalDifficulty` of Ethereum blocks and calculates the estimated merge date by checking how close `totalDifficulty` is to `Terminal Total Difficulty`.
```

Comment:
I think we can include a bit more detail about the usage of average block difficulty and average block time, and how they factored in estimating the merge date.

It was a pretty ingenious approach by you, and I think you can elaborate a bit more. 🙂